__tests__/index.js

```diff
@@ -59,24 +60,43 @@ test('runEveryMinute', async () => {
     expect(fetch).toHaveBeenCalledTimes(0)
 
     await runEveryMinute(getMeta())
-    expect(posthog.capture).toHaveBeenCalledTimes(4)
-    expect(posthog.capture).toHaveBeenCalledWith('Upcoming Invoice', {
-        distinct_id: 'cus_J632IbQFZfXXt5',
-        amount: 150,
-        invoice_date: '02/03/2021',
-        stripe_customer_id: 'cus_J632IbQFZfXXt5',
-        quantity: 0,
-        $set: undefined
-    })
 
-    expect(posthog.capture).toHaveBeenCalledWith('Upcoming Invoice – Above Threshold', {
-        distinct_id: 'cus_J632IbQFZfXXt5',
-        amount: 150,
-        invoice_date: '02/03/2021',
-        stripe_customer_id: 'cus_J632IbQFZfXXt5',
-        alert_threshold: 100,
-        product: undefined,
-        quantity: 0,
-        $set: undefined
-    })
+    const testNumberOfCaptureCalls = () => {
+        expect(posthog.capture).toHaveBeenCalledTimes(5)
+    }
+    testNumberOfCaptureCalls()
+
+    const testUpcomingInvoice = () => {
+        expect(posthog.capture).toHaveBeenCalledWith('Upcoming Invoice', {
+            distinct_id: 'cus_J632IbQFZfXXt5',
+            amount: 150,
+            invoice_date: '02/03/2021',
+            stripe_customer_id: 'cus_J632IbQFZfXXt5',
+            quantity: 0,
+            $set: undefined
+        })
+    }
+    testUpcomingInvoice()
+
+    const testPaidInvoices = () => {
+        const today = new Date()
+        const firstDayThisMonth = new Date(today.getFullYear(), today.getMonth(), 1)
+        const invoicePeriod = firstDayThisMonth.toLocaleDateString('en-GB')
+        expect(posthog.capture).toHaveBeenCalledWith('Paid Invoices', { amount: 0, period: invoicePeriod })
+    }
+    testPaidInvoices()
```

Comment:
is the reason we're calling functions here just readability? wouldn't a comment do the job?

Reply:
Yea just readability – i felt it was getting a little messy, happy to move to a comment

Reply:
i don't _really_ care, was just wondering :D